customer.rb

```diff
@@ -0,0 +1,37 @@
+#class Cusotmer
+require 'csv'
+require 'awesome_print'
+require 'pry'
+
+class Customer
+
+
+  attr_reader :id
+
+  attr_accessor :email, :address
+
+  def initialize(id, email, address = {street: "", city:"", state: "", zip:""})
```

Comment:
Minor stuff:  putting spaces after the colons in the hash keys is more readable.  

Also you can put spaces between the curly braces { } and the content to make them a bit more readable as well.